bin/idam-setup.sh

```diff
@@ -6,34 +6,60 @@ then
 fi
 
 IDAM_URI="http://localhost:5000"
-REDIRECTS=("http://localhost:3001/oauth2/callback" "https://div-pfe-aat.service.core-compute-aat.internal/authenticated")
+
+REDIRECTS=("http://localhost:3001/oauth2/callback" "https://div-pfe-aat.service.core-compute-aat.internal/authenticated" "http://localhost:3000/oauth2/callback")
 REDIRECTS_STR=$(printf "\"%s\"," "${REDIRECTS[@]}")
 REDIRECT_URI="[${REDIRECTS_STR%?}]"
-CLIENT_ID="divorce"
-CLIENT_SECRET=${OAUTH2_CLIENT_SECRET}
+
+DIV_CLIENT_ID="divorce"
+XUI_CLIENT_ID="xuiwebapp"
```

Comment:
Maybe this file could be more re-usable if it were split  and called with different args?

Reply:
sure will do that as a separate PR